app/src/ActionModal/components/ActionModal.vue

```diff
@@ -833,6 +833,10 @@ export default {
             denom,
           }))
         }
+        if (message.rewards) {
```

Comment:
we need this so we can calculate the fees

Reply:
ah, you want to send it to API? It is quite complicated actually, you need `input Validator`, `input FiatValue`... all repeated code, and then the `__typename` thing was giving me errors. Maybe we don't need it

Reply:
ok, thanks for the analysis. will check it and find a solution.